backend/minors/FRE.yaml

```diff
@@ -0,0 +1,72 @@
+---
+type: Minor
+name: French
+code: FRE
+degree:
+urls:
+- https://ua.princeton.edu/fields-study/departmental-majors-degree-bachelor-arts/french-and-italian
+description: Studying the literature, art, society and history of French speakers 
+  around the world allows students to learn directly about two of the richest and most influential
+  cultural traditions in world history (including French literature). It also gives
+  students necessary insight into major historical events and processes, such as nation-building,
+  revolution, colonialism, fascism, migration and democratization, which still resonate and 
+  continue to shape the modern world.
+contacts:
+- type: Director of Undergraduate Studies
+  name: Flora Champy
+  email: fchampy@princeton.edu
+req_list:
+- name: Prerequisites
+  max_counted: 1
+  min_needed: 1
+  explanation: |-
+    To enter the minor in French, students are required to successfully complete the language
+    sequence in French (i.e., FRE 107, 108 or 1027), or to demonstrate the equivalent level 
+    of proficiency. To enter the minor, successful completion of the language sequence is required 
+    by the fall of junior year, but students are strongly encouraged to do so by the spring of their
+    sophomore year. Applications to the minor program are accepted until the end of spring of junior year.
+  completed_by_semester: 5
+  course_list: 
+  - FRE 107
+  - FRE 108
+  - FRE 1027
+- name: Program of Study
+  max_counted: 2
+  min_needed: 5
+  explanation: |- 
+    Students must successfully take a minimum of five courses in the relevant language, linguistics, 
+    literature or culture, excluding courses that do not have a language prerequisite and any 
+    100-level language course. At least three of these courses must be at the 300 level (or higher). 
+    Up to two of these courses may be taken at the 200 level, but FRE 207 and 208 cannot both be counted. 
+    A 200-level course is a prerequisite for taking 300-level courses in French.
+  req_list: 
+  - name: Advanced Level
+    max_counted: 5
+    min_needed: 3
+    course_list: 
+    - FRE 3*
+  - name: Intermediate Level
+    max_counted: 2
+    min_needed: 1
+    course_list: 
+    - FRE 2*
```

Comment:
We can't have both a `course_list` and a `req_list`. Let's put all 2** courses in a requirement, and then put 207 and 208 in the `excluded_work_list`